### PR TITLE
Add release manual

### PR DESCRIPTION
**Pull request by EmmyMiao87**
Change-Id: I47d6ab67500962fc8ddd319dd93af10ac2e1bb3f

## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Doris?
_Put an `x` in the boxes that apply_

- [] Bugfix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)
- [] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)
- [] Code refactor (Modify the code structure, format the code, etc...)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [] I have create an issue on (Fix #ISSUE), and have described the bug/feature there in detail
- [] Compiling and unit tests pass locally with my changes
- [] I have added tests that prove my fix is effective or that my feature works
- [] If this change need a document change, I have updated the document
- [] Any dependent changes have been merged

## Further comments

If this is a relatively large or complex change, kick off the discussion at dev@doris.apache.org by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/zh-CN/community/release-process.md [hunk 1]
```diff
@@ -36,15 +36,19 @@ Apache 的发布必须至少是 IPMC 成员，拥有 apache 邮箱的committer
...
+        1. QA 稳定性测试
+        2. 验证编译镜像正确性
+        3. 准备 Release Nodes
+4. 社区发布投票流程
 	1. 将 tag 打包，签名并上传到[Apache Dev svn 仓库](https://dist.apache.org/repos/dist/dev/incubator/doris)
 	2. 在 [Doris 社区](dev@doris.apache.org)发起投票
 	3. 投票通过后，在Doris社区发 Result 邮件
 	4. 在 [Incubator 社区](general@incubator.apache.org) 发起新一轮投票
 	5. 发 Result 邮件到 general@incubator.apache.org
 4. 完成工作
```
Comment (morningman) [inline docs/zh-CN/community/release-process.md:49]: 5. 完成工作

**Review by morningman — DISMISSED** [review]
Approve

**Review by yangzhg — DISMISSED** [review]
+1

File: docs/en/community/release-process.md [hunk 2]
```diff
@@ -628,7 +672,7 @@ We are pleased to announce the release of Apache Doris 0.9.0-incubating.
 Apache Doris (incubating) is an MPP-based interactive SQL data warehousing for reporting and analysis.
 
 The release is available at:
-http://doris.apache.org/downloads.html
+http://doris.apache.org/master/zh-CN/downloads/downloads.html
```
Comment (morningman) [inline docs/en/community/release-process.md:675]: http://doris.apache.org/master/en/downloads/downloads.html

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**